src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -404,6 +404,11 @@ extension SecureStore: PrivacyPreservingProviding {
 			analyticsSubmitter?.triggerSubmitData()
 		}
 	}
+	
+	var clientMetadata: ClientMetadata? {
+		get { kvStore["clientMetadata"] as ClientMetadata? ?? nil }
+		set { kvStore["clientMetadata"] = newValue }
```

Comment:
```suggestion
		get { kvStore["clientMetadata"] as ClientMetadata? ?? nil }
		set { 
		    kvStore["clientMetadata"] = newValue 
		    analyticsSubmitter?.triggerSubmitData()
		}
```
Every time we *could* send data, try it.